dmscripts/export_framework_applicant_details.py

```diff
@@ -62,10 +63,15 @@ def inner(record):
         supplier_framework = client.get_supplier_framework_info(record['supplier_id'], framework_slug)
         supplier_framework = supplier_framework['frameworkInterest']
         supplier_framework['declaration'] = supplier_framework['declaration'] or {}
+        supplier_framework['countersignedPath'] = supplier_framework['countersignedPath'] or ''
```

Comment:
Hmm, yeah the "blame" on this file is all me but the pattern was a straight-up copy and paste job from somewhere else that does a similar thing with multi-threaded building up of data. I'd rather leave this as is right now, or at least talk through the other approach in person.